conda_forge_tick/update_recipe/version.py

```diff
@@ -134,14 +135,40 @@ def _try_url_and_hash_it(url: str, hash_type: str):
 
 
 def _render_jinja2(tmpl, context):
-    return (
-        jinja2.sandbox.SandboxedEnvironment(undefined=jinja2.StrictUndefined)
-        .from_string(tmpl)
-        .render(**context)
-    )
+    env = jinja2.Environment(undefined=jinja2.StrictUndefined)
```

Comment:
```suggestion
    env = jinja2.sandbox.SandboxedEnvironment(undefined=jinja2.StrictUndefined)
```

This needs to be sandboxed.